ert_storage/compute/misfits.py

```diff
@@ -0,0 +1,32 @@
+import numpy as np
+import pandas as pd
+from uuid import UUID
+from typing import Any, Mapping, List, Optional
+
+
+def _calculate_misfit(
```

Comment:
if these are public functions, wouldnt you remove the _ in front?

Reply:
I think, it's enough when `_calculate_misfits_from_pandas`.